fractions/javaee/undertow/src/main/java/org/wildfly/swarm/undertow/runtime/CertInfoProducer.java

```diff
@@ -47,6 +49,9 @@
     @Singleton
     public CertInfo produceCertInfo() {
         if (generateSelfCertificate.get()) {
+            if (SwarmInfo.isProduct()) {
```

Comment:
Instead of the explicit check, can we just try to call it, and log the happy message if it fails or we can't find it?

That way, when/if WF/EAP supports it, we have zero code-change to make, aside from removing the now-dead code.

With this explicit check, once it's supported upstream, we have to remember to change this.

Else, looks great!